lib/locations/location.ex

```diff
@@ -0,0 +1,19 @@
+defmodule Locations.Location do
+  defstruct vehicle_id: nil,
+            status: nil,
+            stop_id: nil,
+            timestamp: nil,
+            route_id: nil,
+            trip_id: nil,
+            consist: []
```

Comment:
I assume this will change/be added to. I just picked a general set of fields to start us off with